libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2382,3 +2382,16 @@ function dirlist(directoryname) end
 --desc
 --@param filename
 function remove(filename) end
+
+-- Send data over CSRF/ExpressLRS link
+---@param cmd integer -- byte CRSF type
+---@param data string -- byte data to send
+---@return integer -- Number of bytes sent
+function crsfPush(cmd, data) end
```

Comment:
I would prefer to add `crsf` as a table and then `push` as a function. So it would be `crsf:push(cmd, data)`

Reply:
Fine with me, but don't know how to do this.

Reply:
See `CAN` bindings as a example.